packages/grafana-ui/src/components/Forms/Legacy/Select/ButtonSelect.story.internal.tsx

```diff
@@ -1,18 +1,19 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { action } from '@storybook/addon-actions';
-import { withKnobs, object, text } from '@storybook/addon-knobs';
+import { withKnobs, object } from '@storybook/addon-knobs';
```

Comment:
Would be nice to switch this to use controls and add controls for button variant.